src/NuGet.Core/NuGet.ProjectModel/JsonPackageSpecReader.cs

```diff
@@ -35,7 +36,7 @@ public class JsonPackageSpecReader
         /// <param name="packageSpecPath">file path</param>
         public static PackageSpec GetPackageSpec(string name, string packageSpecPath)
         {
-            using (var stream = new FileStream(packageSpecPath, FileMode.Open, FileAccess.Read, FileShare.Read))
+            using (var stream = new FileStream(packageSpecPath, FileMode.Open, FileAccess.Read, share))
```

Comment:
never mind, makes sense now.

---
In reply to: [121762020](https://github.com/NuGet/NuGet.Client/pull/1487#discussion_r121762020) [](ancestors = 121762020)